pkg/storaged/lvol-tabs.jsx

```diff
@@ -110,6 +114,14 @@ function lvol_and_fsys_resize(client, lvol, size, offline, passphrase) {
                 return Promise.reject(_("VDO backing devices can not be made smaller"));
             else
                 return Promise.resolve();
+        } else if (stratis_bdev) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19079-20230811-120908-0b7a3836-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
Coverage of `if` statements is weird. The lines after the `if` (the consequent) are executed, so I don't know how the `if` itself is not executed. We might want filter them all out, like we do with `else` lines.